components/Table/Table.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div>
+    <table>
+      <thead>
+        <tr>
+          <slot name="columns">
+            <th v-for="(head,i) in headers" :key="i">{{ head.text}}</th>
+            <th v-show="showExpand"></th>
+          </slot>
+        </tr>
+      </thead>
+      <tbody>
+        <template v-for="item in items">
+          <tr :key="item.link" @click="expandDetail(item.id)">
+            <slot :row="item">
+              <!-- default content -->
+              <td v-for="(header,i) in headers" :key="i">{{ item[header.value]}}</td>
+              <td v-show="showExpand">                
+                <i class="fas" :class="src(item.id)"></i>
+              </td>
+            </slot>
+          </tr>
+          <tr v-show="showExpand" v-if="toggle.includes(item.id)" :key="item.id">
+            <td :colspan="columnSpan">
+              <slot name="expandItem" :distributions="item.distributions"> 
+                <!-- default content -->
+                <div class="item-background"> Hello world </div>
+              </slot>
+            </td>
+          </tr>
+        </template>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "Table",
+  components: {    
+  },
+  props: {
+    headers: Array,
+    items: Array,
+    showExpand: Boolean,
+    subTitle:String
```

Comment:
this is never used

Reply:
headers use for a display column name and items use for display a row data.
showExpand use for expand and collapse row detail and removed subTitle prop.